xbmc/settings/AdvancedSettings.cpp

```diff
@@ -1440,6 +1440,7 @@ void CAdvancedSettings::SettingOptionsLoggingComponentsFiller(SettingConstPtr se
   list.emplace_back(g_localizeStrings.Get(684), LOGWINDOWING);
   list.emplace_back(g_localizeStrings.Get(685), LOGPVR);
   list.emplace_back(g_localizeStrings.Get(686), LOGEPG);
+  list.emplace_back(IntegerSettingOption(g_localizeStrings.Get(39117), LOGANNOUNCE));
```

Comment:
Explicit `IntegerSettingOption` ctor call can be omitted here. See all the other lines in the method.

Reply:
Thanks, I missed that when I rebased